### ISSUE TITLE
Error trying to run on android

### ISSUE THREAD

**Issue by gabrielsimplicio** [reneweb/react-native-tensorflow#3]
Hi, I'm trying to run on android, but I'm getting the following errors:
```
/home/gabriel/repos/ColorGrain/node_modules/react-native-tensorflow/android/src/main/java/com/rntensorflow/RNTensorFlowInferenceModule.java:70: error: cannot find symbol
          float[] srcData = readableArrayToFloatArray(data.getArray("data"));
                            ^
  symbol:   method readableArrayToFloatArray(ReadableArray)
  location: class RNTensorFlowInferenceModule
/home/gabriel/repos/ColorGrain/node_modules/react-native-tensorflow/android/src/main/java/com/rntensorflow/RNTensorFlowInferenceModule.java:73: error: cannot find symbol
          int[] srcData = readableArrayToIntArray(data.getArray("data"));
                          ^
  symbol:   method readableArrayToIntArray(ReadableArray)
  location: class RNTensorFlowInferenceModule
/home/gabriel/repos/ColorGrain/node_modules/react-native-tensorflow/android/src/main/java/com/rntensorflow/RNTensorFlowInferenceModule.java:79: error: cannot find symbol
          int[] srcData = readableArrayToIntArray(data.getArray("data"));
                          ^
  symbol:   method readableArrayToIntArray(ReadableArray)
  location: class RNTensorFlowInferenceModule
/home/gabriel/repos/ColorGrain/node_modules/react-native-tensorflow/android/src/main/java/com/rntensorflow/RNTensorFlowInferenceModule.java:82: error: cannot find symbol
          byte[] srcData = readableArrayToByteBoolArray(data.getArray("data"));
                           ^
  symbol:   method readableArrayToByteBoolArray(ReadableArray)
  location: class RNTensorFlowInferenceModule
/home/gabriel/repos/ColorGrain/node_modules/react-native-tensorflow/android/src/main/java/com/rntensorflow/RNTensorFlowInferenceModule.java:85: error: cannot find symbol
          byte[] srcData = readableArrayToByteStringArray(data.getArray("data"));
                           ^
  symbol:   method readableArrayToByteStringArray(ReadableArray)
  location: class RNTensorFlowInferenceModule
5 errors
:react-native-tensorflow:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.
```
I'm trying to run on ubuntu

**Comment by reneweb** [comment]
Ah, for some reason this file didn't got checked in with those changes :/

**Comment by reneweb** [comment]
I just published 0.0.11 which should fix this, I will also look into adding a travis integration to at least compile everything on push, so we avoid this in the future

**Comment by reneweb** [comment]
Sorry, Just had to update to 0.0.12 with similar issue
